Extension/src/LanguageServer/client.ts

```diff
@@ -382,6 +390,14 @@ class DefaultClient implements Client {
         }
     }
 
+    public onDidChangeVisibleTextEditors(editors: vscode.TextEditor[]): void {
+        //Apply text decorations to inactive regions
+        for (let e of editors) {
+            let valuePair: [vscode.TextEditorDecorationType, vscode.Range[]] = this.inactiveRegionsDecorations.get(e.document.uri.toString());
+            e.setDecorations(valuePair[0], valuePair[1]); // VSCode clears the decorations when the text editor becomes invisible
```

Comment:
need to guard against valuePair being undefined here.

Reply:
Guard has been brought back in + using `if (valuePair)` instead of `if (valuePair !== undefined)`